demo/admin/src/common/blocks/HeadlineBlock.tsx

```diff
@@ -22,16 +23,9 @@ export const HeadlineBlock = createCompositeBlock(
         displayName: "Headline",
         blocks: {
             eyebrow: {
-                block: createCompositeSetting<HeadlineBlockData["eyebrow"]>({
-                    defaultValue: undefined,
-                    AdminComponent: ({ state, updateState }) => (
-                        <BlocksFinalForm<Pick<HeadlineBlockData, "eyebrow">>
-                            onSubmit={({ eyebrow }) => updateState(eyebrow)}
-                            initialValues={{ eyebrow: state }}
-                        >
-                            <Field name="eyebrow" label="Eyebrow" component={FinalFormInput} fullWidth />
-                        </BlocksFinalForm>
-                    ),
+                block: createCompositeBlockTextField({
+                    defaultValue: "",
+                    displayName: "Eyebrow",
```

Comment:
We could also use the `title` prop in blocks.eyebrow in `createCompositeBlock`. What do you think?

Reply:
How do you mean that?

Reply:
Like this:

```ts
eyebrow: {
  block: createCompositeBlockTextField({ defaultValue: "" }),
  title: "Eyebrow"
}
```

Reply:
but then there would be no text that describes the textfield. Or i would have to map title to label => `label={title}`

Reply:
I guess both is fine 👍🏼